packages/jsActions/nanoflow-actions-native/src/geolocation/GetCurrentLocation.ts

```diff
@@ -84,8 +101,8 @@ export async function GetCurrentLocation(
             if (position.coords.heading != null && position.coords.heading !== -1) {
                 mxObject.set("Heading", new Big(position.coords.heading.toFixed(8)));
             }
-            if (position.coords.speed != null) {
-                mxObject.set("AltitudeAccuracy", new Big(position.coords.speed.toFixed(8)));
+            if (position.coords.speed != null && position.coords.speed !== -1) {
```

Comment:
What's the meaning of this check on `-1` exactly, going backwards? 🤔

Reply:
Sometimes the api retrieves -1 in the numbers when its not available. Same for heading and other fields.

Reply:
🤔